### ISSUE TITLE
RCS1190 doesn't understand the verbatim identifier (@)

### ISSUE THREAD

**Issue by fgimian** [JosefPihrt/Roslynator#835]
**Product and Version Used**: 3.2.0

**Steps to Reproduce**:

1. Create a console application using .NET 5
2. Install Rosylnator.Analyzers
3. Update Program.cs to the following

```csharp
using System;

namespace RoslynatorProblem
{
    internal static class Program
    {
        internal static void Main()
        {
            string longText =
                @"Long line with \ and other crazy things " +
                @"causes \Roslynator to get confused";
            longText += " and that's bad";
            Console.WriteLine(longText);
        }
    }
}
```

**Actual Behavior**:

Roslynator complains about RCS1190, Join string expressions.

**Expected Behavior**:

No errors should be reported.

This occurs when you join multiple verbatim strings using the `@` prefix only.  It seems that Roslynator doesn't understand this.

**Comment by JosefPihrt** [comment]
Hi,

I'm not sure that I understand your suggestion. Are you saying that the diagnostic should not be reported if the expressions that should be joined span over multiple lines?

**Comment by fgimian** [comment]
> 
> 
> Hi,
> 
> I'm not sure that I understand your suggestion. Are you saying that the diagnostic should not be reported if the expressions that should be joined span over multiple lines?

Precisely, and in fact Roslynator indeed does not report this when a string spans multiple lines without the `@` prefix.

In case the reasons why this is a good idea are required, here we go 😄 

* Limiting line lengths is always a good idea as enforced by RCS0056
* Often long strings do exist in code, especially tests
* Using `+` is the only method that works with string constants, `string.Concat` won't work for constants

Ultimately, Roslynator totally respects this when joining regular strings or even those with interpolation, but simply not verbatim strings.

Happy to provide further examples if need be.

**Comment by JosefPihrt** [comment]
I got it, thanks.

And I'm really glad that you like Roslynator!
